src/content/docs/apm/agents/php-agent/php-agent-api/newrelic_set_appname.mdx

```diff
@@ -37,7 +37,7 @@ For other app-naming options, see [Name your PHP application](/docs/agents/php-a
 
 ### Call and location behavior
 
-New Relic highly recommends that you call this function as soon as possible and in as shallow of a call stack as possible. Applications are discrete entities in APM, and discontinuity of transaction traces is inherent to this method's usage. After connecting with the new app name, the new transactions start reporting to the new application without the context of the previous application or the data that has been reported to it.
+We highly recommend that you call this function as soon as possible after you start monitoring your app and in as shallow of a call stack as possible. Applications are discrete entities in APM, and when you change an app's name, there's nothing to tie the data of the two app names together. This can lead to a discontinuity of transaction traces. After connecting with the new app name, the new transactions start reporting to the new application without the context of the previous application or the data that has been reported to it.
```

Comment:
Is it worth going into what a shallow call stack means? Or should we just assume that someone coming to this section will understand what that means?

Reply:
Yeah, I was trying to skate around this a little bit @homelessbirds because I'm not sure what it means... :D